Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Diagnostics/DiagnosticDescriptors.cs

```diff
@@ -38,5 +36,13 @@ public static class DiagnosticDescriptors
             category: "AWSLambdaCSharpGenerator",
             DiagnosticSeverity.Info,
             isEnabledByDefault: true);
+        
+        public static readonly DiagnosticDescriptor MissingDependencies = new DiagnosticDescriptor(id: "AWSLambda0104",
+            title: "Missing reference to a required dependency",
+            messageFormat: "Your project has a missing required package dependency. Please add a reference to the following package: {0}",
+            category: "AWSLambdaCSharpGenerator",
+            DiagnosticSeverity.Error,
+            isEnabledByDefault: true);
```

Comment:
This needs to be added in https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Diagnostics/AnalyzerReleases.Unshipped.md

There are instructions on the top, AWSLambda0104 would like similar to [these](https://github.com/aws/aws-lambda-dotnet/blob/02aaa44aa35ddf36775ec97a06bc9cac1460e228/Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Diagnostics/AnalyzerReleases.Unshipped.md)

Reply:
Done